pkg/kube/exec.go

```diff
@@ -147,7 +148,8 @@ func execute(method string, url *url.URL, config *restclient.Config, stdin io.Re
 	if err != nil {
 		return err
 	}
-	return exec.Stream(remotecommand.StreamOptions{
+	// Get context from a caller function. Ticket to track: https://kasten.atlassian.net/browse/K10-15138
```

Comment:
Please remove this link. Create a github issue and link that instead

Reply:
fixed